src/core/Akka/Actor/Props.cs

```diff
@@ -781,7 +782,12 @@ private DynamicProps(Props copy, Func<TActor> invoker)
         protected override Props Copy()
         {
             Props initialCopy = base.Copy();
+#if CLONEABLE
             var invokerCopy = (Func<TActor>)invoker.Clone();
+#else
+            // TODO: CORECLR FIX IT
+            var invokerCopy = (Func<TActor>)invoker;
```

Comment:
Not that it'll help you yet, but this is being restored in netstandard2.0 (dotnet/corefx#12251).